cmd/snap/error.go

```diff
@@ -100,6 +100,12 @@ func errorToCmdMessage(snapName string, e error, opts *client.SnapOptions) (stri
 		// arch/channel/revision. Surface that here somehow!
 
 		msg = i18n.G("snap %q not found")
+		if snapName == "" {
```

Comment:
I feel a little bit uneasy about this, its clever but `snapName` seems to be missleading now. But maybe I don't see the big picture. Maybe a comment where about err.Value and why it is used as snapName in this context and what sets it?